### PR TITLE
Feature/1 initialize operator

### LINKED ISSUE #1: Initialize backup-operator
- create CRD
- fix build pipeline
- fix makefile
- release as helm chart

### PR DESCRIPTION
**Pull request by jelemux**
Resolves #1

### CODE REVIEW

File: pkg/api/ecosystem/backupRestClient.go [hunk 1]
```diff
@@ -0,0 +1,251 @@
+package ecosystem
```
Comment (cesmarvin) [inline pkg/api/ecosystem/backupRestClient.go:1]: 🚫 **[checkstyle]** <dupl> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>1-251 lines are duplicate of `pkg/api/ecosystem/restoreRestClient.go:1-251`

File: pkg/api/ecosystem/restoreRestClient.go [hunk 2]
```diff
@@ -0,0 +1,251 @@
+package ecosystem
```
Comment (cesmarvin) [inline pkg/api/ecosystem/restoreRestClient.go:1]: 🚫 **[checkstyle]** <dupl> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>1-251 lines are duplicate of `pkg/api/ecosystem/backupRestClient.go:1-251`

File: pkg/api/ecosystem/backupRestClient.go (hunk 1 above)
Comment (jelemux) [reply pkg/api/ecosystem/backupRestClient.go:1]: I'm inclined to ignore this.

File: pkg/api/ecosystem/restoreRestClient.go (hunk 2 above)
Comment (jelemux) [reply pkg/api/ecosystem/restoreRestClient.go:1]: I'm inclined to ignore this.

**Pull request merged**